src/samples/aspnet/Elsa.Samples.AspNet.Tenants.External/Program.cs

```diff
@@ -0,0 +1,143 @@
+using System.Security.Claims;
+using Elsa;
+using Elsa.EntityFrameworkCore.Common;
+using Elsa.EntityFrameworkCore.Extensions;
+using Elsa.EntityFrameworkCore.Modules.Management;
+using Elsa.EntityFrameworkCore.Modules.Runtime;
+using Elsa.Extensions;
+using Elsa.Tenants.Extensions;
+using Elsa.Tenants.Helpers;
+using FastEndpoints.Swagger;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using NSwag;
+using NSwag.AspNetCore;
+using NSwag.Generation.Processors.Security;
+
+WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
+ConfigurationManager configuration = builder.Configuration;
+
+var identitySection = configuration.GetSection("Identity");
+
+// Add services to the container.
+builder.Services.AddControllers();
+builder.Services.AddElsa(elsa =>
+{
+    ElsaDbContextOptions dbContextOptions = EfCoreDbContextBuilder.BuildDbContextTenantOption();
+    string postgresConnectionString = configuration.GetConnectionString("Postgres")!;
```

Comment:
I think it would be user-friendly to provide examples using SQLite, as it doesn't necessitate any external dependencies. This approach could make the initial setup and learning curve easier for users. Of course, we should continue to use PostgreSQL in examples specifically designed to demonstrate its usage. This way, we can cater to both simplicity and specificity in our examples.